vpr/src/route/router_lookahead_map.cpp

```diff
@@ -330,11 +330,22 @@ std::pair<float, float> MapLookahead::get_expected_delay_and_cong(RRNodeId from_
 
         auto from_ptc = rr_graph.node_ptc_num(from_node);
 
-        std::tie(expected_delay_cost, expected_cong_cost) = util::get_cost_from_src_opin(src_opin_delays[from_layer_num][from_tile_index][from_ptc][to_layer_num],
-                                                                                         delta_x,
-                                                                                         delta_y,
-                                                                                         to_layer_num,
-                                                                                         get_wire_cost_entry);
+        /* get_cost_from_src_opin iterates over all routing segments passed to it (the first argument) and returns 
```

Comment:
I think you need to give some high-level context.  
/* We could reach the sink by using an intermediate wire on any reachable layer. We consider all these options and return the minimum cost one.